plotter/heat.go

```diff
@@ -178,16 +178,24 @@ func (h *HeatMap) DataRange() (xmin, xmax, ymin, ymax float64) {
 		xmax = 0.5
 		xmin = -0.5
 	default:
+		// Min and max are the first and last h.GridXYZ.X
+		// values extended beyond by half the distance
+		// between them and their adjacent h.GridXYZ.X
+		// value.
 		xmax = (3*h.GridXYZ.X(c-1) - h.GridXYZ.X(c-2)) / 2
```

Comment:
Yeah. It's sort of habit for me to reduce the number of interface method calls. This doesn't get called in a loop, so probably it should be expanded.